### ISSUE TITLE
client version 1.44 is too new

### ISSUE THREAD

**Issue by tcurdt** [aacebedo/dnsdock#116]
I assume this needs a new release:

```
docker run -v /var/run/docker.sock:/var/run/docker.sock --name dnsdock -p 172.17.0.1:53:53/udp aacebedo/dnsdock:latest-amd64
13:06:38.249 | CRITICAL ▶ Error: 'Error getting containers: Error response from daemon: {"message":"client version 1.44 is too new. Maximum supported API version is 1.43"}'
```

**Comment by tcurdt** [comment]
Seems like this fork was some good work https://github.com/frenchzed/dnsdock

**Comment by aacebedo** [comment]
PR has been merged and release v1.18.0-rc2 done
